frontend/src/pages/modelServing/screens/projects/nimUtils.ts

```diff
@@ -77,3 +80,44 @@ export const getNIMServingRuntimeTemplate = async (
     return undefined;
   }
 };
+
+export const updatePVCName = (
```

Comment:
```suggestion
export const updateServingRuntimeTemplate = (
```

I think it's more explicit to say you're updating the template -- not the PVC Name -- as you're applying the pvc name to the template.

Reply:
Updated